client/src/components/GraphViewerComponent/GraphViewerRelationshipCreateComponent/GraphViewerRelationshipCreateComponent.js

```diff
@@ -93,56 +106,98 @@ export class GraphViewerRelationshipCreateComponent extends Component {
       document.getElementById("create-relationship-heading").focus();
     }, 200);
 
-    const { selectedNode, selectedNodes } = this.props;
-    const sourceModelId = selectedNodes.find(x => x.id !== selectedNode.id).modelId;
-    const targetModelId = selectedNode.modelId;
-
     try {
-      const relationshipItems = await new ModelService().getRelationships(sourceModelId, targetModelId);
+      const { selectedNode, selectedNodes } = this.props;
+      const source = selectedNodes.find(x => x.id !== selectedNode.id);
+      const target = selectedNode;
+      this.setState({sourceId: source.id});
+      this.setState({targetId: target.id});
```

Comment:
Can you please put ` ` chars on either side of the property? E.g. `({ sourceId: source.id })`

Reply:
in addition - if you run `npm run lint -- --fix` it can automatically amend your code with formatting etc.

Reply:
Try running the script but the formatting didn't change.
Changed the formatting manually and will make sure be more aware of this.

Reply:
Great. Let's also combine these two lines (i.e. set both sourceId and targetId in the same call to setState).

Reply:
Ready